src/branding.js

```diff
@@ -0,0 +1,38 @@
+// @flow
```

Comment:
I had the same experience, and then I got used to that. I makes the source to look like a different language.